catalog/hierarchy/README.md

```diff
@@ -0,0 +1,35 @@
+# Org hierarchy blueprint
+
+[TOC]
```

Comment:
Is this expected to be rendered? I don't see it rendered in preview

Reply:
It looks like GitHub does the TOC automatically, so I'll remove this: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/